### PR TITLE
Added ISE reference to workflow

### PR DESCRIPTION
**Pull request by visharm**

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @visharm , please follow these [instructions](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec), and make sure that:
- The `eng\mgmt\mgmtmetadata\<rpName>.txt` file is included in this PR
- Versions have been updated, if necessary
- Package tags, release notes, have been updated

And please put a link to the corresponding swagger review PR in comments. Thanks

**Comment by visharm** [comment]
https://github.com/Azure/azure-rest-api-specs/pull/6601

File: sdk/logic/Microsoft.Azure.Management.Logic/src/Microsoft.Azure.Management.Logic.csproj [hunk 1]
```diff
@@ -12,6 +12,7 @@
 		<![CDATA[
 	  - Bug fix for updating Logic Apps that have an attached Integration Account
 	  - Added WorkflowRunActionRequestHistories, WorkflowRunActionRepetitionsRequestHistories
+	  - Added Integration Service Environment reference in Workflow
```
Comment (isra-fel) [inline sdk/logic/Microsoft.Azure.Management.Logic/src/Microsoft.Azure.Management.Logic.csproj]: I think you'd better remove the notes from last release

Comment (isra-fel) [reply sdk/logic/Microsoft.Azure.Management.Logic/src/Microsoft.Azure.Management.Logic.csproj]: And please also update `<Version>` in this file

File: sdk/logic/Microsoft.Azure.Management.Logic/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
-[assembly: AssemblyVersion("4.1.0.0")]
+[assembly: AssemblyVersion("4.2.0.0")]
 [assembly: AssemblyFileVersion("4.0.0.0")]
```
Comment (isra-fel) [inline sdk/logic/Microsoft.Azure.Management.Logic/src/Properties/AssemblyInfo.cs]: `AssemblyFileVersion` should be updated

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/logic/Microsoft.Azure.Management.Logic/src/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (visharm) [reply sdk/logic/Microsoft.Azure.Management.Logic/src/Properties/AssemblyInfo.cs]: Could you please take a look at the CR?

**Comment by isra-fel** [comment]
@visharm , you are making a big change and updating API versions, so please
1. Update major version
1. Update the api tag in `AzSdk.RP.props`

Please also fix test errors -- they are mainly caused by the change of the API

**Comment by isra-fel** [comment]
Hello @visharm any update?

**Comment by isra-fel** [comment]
Closing this for now @visharm, feel free to reopen to continue, thanks

**Pull request closed without merging**